guides/common/modules/proc_adding-amazon-ec2-details-to-a-compute-profile.adoc

```diff
@@ -15,5 +15,12 @@ If you have a VPC for provisioning new hosts, use its subnet.
 . Click *Submit* to save the compute profile.
 
 .CLI procedure
-The compute profile CLI commands are not yet implemented in {ProjectName}.
-As an alternative, you can include the same settings directly during the host creation process.
+. To add Amazon EC2 details to a compute profile, use the following command:
```

Comment:
Us docs people are huge fans of consistency :) Notice how at some places you're using the wording "use the following command" and in others "enter the following command". It's not a big issue, but I still can't help pointing it out.

Reply:
Writing out "use the following command" does not add any value; users can see that the whole step is about using a command. A more helpful way to document commands is to describe what those commands do. A good example of this approach is section [2.12. Creating an installation medium for Ubuntu 22.04](https://theforeman-foreman-documentation-preview-pr-2988.surge.sh/nightly/Provisioning_Hosts/index-foreman-el.html#Creating_an_Installation_Medium_for_Ubuntu_22_04_provisioning): notice how each of the steps in that procedure talk about _what_ that command does while avoiding words like "use this command to [...]".

Can you think of a way to rephrase line 18 (and the other steps in your PR) to be more descriptive like this?